schema/defs.yaml

```diff
@@ -4,6 +4,9 @@ title: Overture Maps Feature Schema Common Definitions
 description: Common schema definitions shared by all themes
 "$defs":
   propertyDefinitions:
+    id:
+      type: string
+      description: A feature ID. This may be an ID associated with the Global Entity Reference System (GERS) if—and-only-if the feature represents an entity that is part of GERS.
```

Comment:
How do we know whether an ID is associated with the Global Entity Reference System?